corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -43,6 +48,9 @@ def add_custom_parameter(key, value):
 # for rapid iteration on fixtures/cases/etc.
 INITIAL_SYNC_CACHE_THRESHOLD = 60  # 1 minute
 
+# Max amount of bytes to have in memory when streaming a file
+MAX_BYTES = 1000000  # 1MB
```

Comment:
Don't really have a good idea for how much of the file we should read into memory. Any insight would be helpful